examples/platform/efr32/init_otSystem.c

```diff
@@ -102,6 +112,8 @@ void initOtSysEFR(void)
     efr32RadioInit();
     efr32AlarmInit();
     efr32MiscInit();
+#if defined(EFR32MG12)
     efr32RandomInit();
```

Comment:
Is this function not available on EFR32MG21?   If so, should this perhaps be `#ifndef EFR32MG21`

Reply:
This function doesn't exist for the MG21 it has to do with hardware crypto acceleration inits and the mg21 doesn't have that feature. I will change for ifndef